alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ChannelDistributionUIConfig.java

```diff
@@ -70,6 +79,9 @@ public ChannelDistributionUIConfig(ChannelKey channelKey, String label, String u
         ConfigField channelName = new EndpointSelectField(KEY_CHANNEL_NAME, LABEL_CHANNEL_NAME, DESCRIPTION_CHANNEL_NAME)
                                       .applyClearable(false)
                                       .applyRequired(true);
+        if (null != channelConfigValidator) {
+            channelName.applyValidationFunctions(channelConfigValidator);
```

Comment:
We might want to change this variable to `channelNameField` so it doesn't appear to be a string.